CMakeLists.txt

```diff
@@ -137,6 +139,9 @@ if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY AND NOT WIN32)
   )
 endif()
 
+# Enable documentation build
+option(BUILD_DOCS "Build documentation" ON)
```

Comment:
I don't recall the standard... but all rocFFT specific build option should start with ROCFFT_, do we? Unless it is a rocm cmake level stuff.

Reply:
I would think we want to make similar changes to all math libs, not just rocFFT.  So IMO the generic name makes more sense.

Reply:
This will be a ROCm wide build option.

Reply:
BUILD_DOCS should be OFF by default

Reply:
Updated.